gems/spree_auth_devise/GHSA-6mqr-q86q-6gwr.yml

```diff
@@ -0,0 +1,35 @@
+---
+gem: spree_auth_devise
+ghsa: 6mqr-q86q-6gwr
+url: https://github.com/solidusio/solidus_auth_devise/security/advisories/GHSA-xm34-v85h-9pg2
```

Comment:
This advisory is for `solidus_auth_devise`, but the gem name is `spree_auth_devise`.

Reply:
Solidus is a popular fork of Spree, so they share a lot of dependencies

Reply:
Already included in gems/spree_auth_devise/CVE-2021-41275.yml